package.json

```diff
@@ -69,7 +69,7 @@
         "eslint-plugin-simple-import-sort": "^10.0.0",
         "fast-xml-parser": "^4.0.11",
         "fs-extra": "^9.1.0",
-        "glob": "latest",
```

Comment:
Thank you! ❤️ I was once traveling through the git history back and I wasn't able to build the old commit because this wasn't pinned.

You still have a couple of other dependencies that are not pinned so this could still happen in the future with any of them. What do you think about pinning all of them?

Reply:
Yes, I'll probably do this. I already send PRs to update deps in the repo when something major is released so I'm not concerned about not moving forward.

That being said, the bisect script I use (which I'll post when I'm not replying on my phone...) detects when something like this is broken and can skip the commit.

Reply:
Here's the script I use with `git bisect run`:

```sh
#!/bin/zsh

COMMAND=gulp
if [ -f Herebyfile.mjs ]; then
    COMMAND=hereby
fi

# Make sure that tests are going to run at all
$COMMAND runtests -t "notarealtest" --no-typecheck --no-lint &> /dev/null || npm ci &> /dev/null
$COMMAND runtests -t "notarealtest" --no-typecheck --no-lint &> /dev/null || exit 125

$COMMAND runtests -t "testme" --no-lint --no-typecheck &> /dev/null && exit 0 || exit 1
```

That being said, if you're going way way back in time, try out `npm install --before`; you can give it a date and it will give you the dependencies as of that date. It works most of the time.